app/javascript/components/public_health/PatientsTable.js

```diff
@@ -719,27 +722,30 @@ class PatientsTable extends React.Component {
                   )}
                 </InputGroup>
               </Form>
-              <CustomTable
-                dataType="patients"
-                columnData={this.state.table.displayedColData}
-                rowData={this.state.table.rowData}
-                totalRows={this.state.table.totalRows}
-                handleTableUpdate={query => this.updateTable({ ...this.state.query, order: query.orderBy, page: query.page, direction: query.sortDirection })}
-                handleSelect={this.handleSelect}
-                handleEntriesChange={this.handleEntriesChange}
-                handlePageUpdate={this.handlePageUpdate}
-                getRowCheckboxAriaLabel={this.getRowCheckboxAriaLabel}
-                isSelectable={true}
-                isEditable={false}
-                isLoading={this.state.loading}
-                page={this.state.query.page}
-                selectedRows={this.state.selectedPatients}
-                selectAll={this.state.selectAll}
-                entryOptions={this.state.entryOptions}
-                entries={parseInt(this.state.query.entries)}
-                orderBy={this.state.query.order !== undefined ? this.state.query.order : ''}
-                sortDirection={this.state.query.direction !== undefined ? this.state.query.direction : ''}
-              />
+              <div className="patients-table">
+                <CustomTable
+                  dataType="patients"
+                  columnData={this.state.table.displayedColData}
+                  rowData={this.state.table.rowData}
+                  totalRows={this.state.table.totalRows}
+                  handleTableUpdate={query => this.updateTable({ ...this.state.query, order: query.orderBy, page: query.page, direction: query.sortDirection })}
+                  handleSelect={this.handleSelect}
+                  handleEntriesChange={this.handleEntriesChange}
+                  handlePageUpdate={this.handlePageUpdate}
+                  getRowCheckboxAriaLabel={this.getRowCheckboxAriaLabel}
+                  isSelectable={true}
+                  isEditable={false}
+                  isLoading={this.state.loading}
+                  page={this.state.query.page}
+                  selectedRows={this.state.selectedPatients}
+                  selectAll={this.state.selectAll}
+                  entryOptions={this.state.entryOptions}
+                  entries={parseInt(this.state.query.entries)}
+                  orderBy={this.state.query.order !== undefined ? this.state.query.order : ''}
+                  sortDirection={this.state.query.direction !== undefined ? this.state.query.direction : ''}
+                  getRowClassName={this.getRowClassName}
```

Comment:
this is SO picky but can you put this above the `getRowCheckboxAriaLabel` since they are related?